qiskit_nature/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_ucc_factory.py

```diff
@@ -173,6 +184,18 @@ def initial_state(self, initial_state: Optional[QuantumCircuit]) -> None:
         the :class:`~.HartreeFock`."""
         self._initial_state = initial_state
 
+    @property
+    def callback(self) -> Optional[Callable[[int, np.ndarray, float, float], None]]:
+        """Returns the callback."""
+        return self._vqe._callback
+
+    @callback.setter
+    def callback(self, callback: Optional[Callable[[int, np.ndarray, float, float], None]]) -> None:
+        """Sets the callback."""
+        if callback is None:
+            return
+        self._vqe._callback = callback
```

Comment:
Similar comment again for None - if you don't want the prior callback to be done then passing in None should "reset" it so no callback is called. Prviate member access too as note in getter above.